build.gradle

```diff
@@ -83,7 +82,10 @@ allprojects {
     }
 
     apply from: rootProject.file('gradle/dokka.gradle')
-    apply from: rootProject.file('gradle/publish.gradle')
+
+    if (!rootProject.ext.misc.contains(project.name)) {
```

Comment:
This `misc` is unclear. Wouldn't it better to rename it to clarify what kind of projects are included?